docs/hardware-interfaces/gpio.md

```diff
@@ -2,20 +2,62 @@
 title: GPIO
 ---
 
-# GPIO
+GPIOs (General Purpose Input Outputs) are user-controllable pins that can be used to interact with external circuits.
 
-The GPIOs can be accessed through the GPIO sysfs interface, see the [documentation](https://www.kernel.org/doc/Documentation/gpio/sysfs.txt) for details. 
+Let's get started 🚀
 
-> The GPIO sysfs interface is deprecated but is currently the best option for userspace GPIO access. More context available in [this post by Luz on the Onion Community](https://community.onion.io/topic/4892/can-bus-using-mcp2515-with-omega2/13).
+## Hardware
 
-An important note: because of changes in the kernel, the GPIOs are not numbered like they were before:
+Table listing number of GPIOs on Omega2/2+ vs Omega2S/Omega2S+
```

Comment:
Please update this to be a simple table that shows the number of GPIOs available on the Omega2/2+ vs Omega2S/Omega2S+